dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckReport.java

```diff
@@ -17,16 +17,29 @@
 package com.google.cloud.tools.opensource.classpath;
 
 import com.google.auto.value.AutoValue;
+import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableList;
 
 /**
  * The result of a static linkage check.
  */
 @AutoValue
-abstract class StaticLinkageCheckReport {
+public abstract class StaticLinkageCheckReport {
+
   abstract ImmutableList<JarLinkageReport> getJarLinkageReports();
 
-  static StaticLinkageCheckReport create(Iterable<JarLinkageReport> jarLinkageReports) {
+  @VisibleForTesting
```

Comment:
I'm not really sure this is true. But if it is, it's true for the other factory methods, and in fact the other value types entirely. Not sure it's worth it to add.